src/modules/add_task.js

```diff
@@ -0,0 +1,17 @@
+import Task, { addToLocalStorage } from './local_storage.js';
+
+import showActivity from './show_activity.js';
+
+const taskLists = JSON.parse(localStorage.getItem('data'));
+
+const addTask = () => {
```

Comment:
- [ ] Great job implementing the changes requested by the previous reviewer. But, I think the changes you have made caused more issues. For example, when I tried to add a task it did not work properly. Below you can see the screenshot please fix the issue then request another review.
![Screenshot (194)](https://user-images.githubusercontent.com/67087664/154672278-2e1e39ca-e1b0-4f5d-b1b2-03beb14da7c3.png)